deployments/local-minio/deploy.sh

```diff
@@ -6,73 +6,68 @@ name="cdr"
 helptext="\
 Usage: deploy.sh [command]
 Utility script for setting up a docker network and deploying a local instance of CDR into a docker swarm.
-'deploy' is the default option if no command is given.
+'update' is the default option if no command is given.
 
 Commands:
-  d, deploy\t\tcreate the docker network and stack, stopping if either already exists
-  u, update\t\tattempts an in-place update of the deployed docker stack
-  c, clean\t\tremoves the deployed docker stack and network (destructive)
-  r, redeploy\t\tclean + deploy: completely removes the stack and network and deploys new instances (destructive)
+  u, update\t\tUpdate deployed stack. Fastest option, but does not pick up all changes. Create network if necessary. Exit with error if the network exsists, but is not properly configured.
+  r, redeploy\t\tStop existing stack. Picks up all changes except changes to the overlay network. Create the docker network (if necessary) and deploy stack. Exit with error if the network exsists, but is not properly configured.
+  c, clean\t\tRemove the deployed stack and network.
```

Comment:
The changes to the description of the commands is a good summary of what this PR actually does.